src/app/pages/organizations/developers/developer/developer-view.jsx

```diff
@@ -94,10 +119,10 @@ function ChplDeveloperView({ dispatch }) {
 
   return (
     <Box className={classes.mainContent}>
-      <Box className={state === 'editUser' ? classes.fullWidthGridRow : ''}>
+      <Box className={`${classes.lefthandContainer} ${state === 'editUser' ? classes.fullWidthGridRow : ''}`}>
```

Comment:
What happens when someone is editing a user? In the original code it should have made the edit panel full width, but I don't think that'll happen here

Reply:
Working this out